ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -142,6 +142,9 @@ public boolean addTransactionHash(final Hash transactionHash) {
 
   public ValidationResult<TransactionInvalidReason> addLocalTransaction(
       final Transaction transaction) {
+    if (transaction.getChainId().isEmpty() && !configuration.isUnprotectedTransactionsAllowed()) {
```

Comment:
Such a large test impact for such a little 3 line change ;)

Reply:
TIL you can't do in-line emojis in github comments.

Reply:
it was a bit daunting yeah.